sdk/spring/spring-cloud-azure-resourcemanager/pom.xml

```diff
@@ -46,6 +46,12 @@
       <version>2.30.0</version> <!-- {x-version-update;com.azure.resourcemanager:azure-resourcemanager;dependency} -->
     </dependency>
 
+    <dependency>
+      <groupId>com.azure.spring</groupId>
+      <artifactId>spring-cloud-azure-stream-binder-servicebus-core</artifactId>
+      <version>4.12.0-beta.1</version> <!-- {x-version-update;com.azure.spring:spring-cloud-azure-stream-binder-servicebus-core;current} -->
+    </dependency>
```

Comment:
This is not correct, we should not depend on the stream binder in the resource manager lib.